src/main/java/Main.java

```diff
@@ -24,27 +25,38 @@ public static void main(String[] args) {
             System.out.println(e);
```

Comment:
Where does this `e` variable come from?
Can you also replace `System.out.println()`by a logger ?

Reply:
@ebenjs hi! variable `e` comes from catching an error of type `Exception` declared on line 23